kpatch-build/kpatch-build

```diff
@@ -451,6 +451,12 @@ find_kobj() {
 	done
 }
 
+check_pahole_supports_btf() {
+	PAHOLE_SUPPORTS_BTF=0
+	cp "$TOOLSDIR"/create-diff-object "$TEMPDIR"
+	pahole -J "$TEMPDIR"/create-diff-object && PAHOLE_SUPPORTS_BTF=1
```

Comment:
This check is not enough. pahole may support -J but still have an issue with big number of sections. See commit message for https://github.com/dynup/kpatch/commit/e25450f5a80349b88607634a80a3fa4e02823caa for more info.

Anyway kpatch builds do not benefit from this at all, so instead of complicated error-prone checks of pahole state we can always disable this.